### PR TITLE
Add i18n items for the PipelineRun role

### PR DESCRIPTION
**Pull request by LinuxSuRen**
### What type of PR is this?

/area devops
/cc @kubesphere/sig-devops 
/cherrypick release-3.2

### What this PR does / why we need it:
A new CRD PipelineRun is introduced in v3.2. We need the i18n items for that.

### Which issue(s) this PR fixes:

related issue https://github.com/kubesphere/ks-devops/issues/383

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note
None
```

### Additional documentation, usage docs, etc.:

```docs
None
```

### CODE REVIEW

**Review by JohnNiang — COMMENTED** [review]
Hi @LinuxSuRen , could you tell us exactly where to use them?

**Comment by LinuxSuRen** [comment]
> Hi @LinuxSuRen , could you tell us exactly where to use them?

You will see it when you try to create a new role, and set the permission of DevOps for it.

/cc @kubesphere/sig-console

**Review by JohnNiang — APPROVED** [review]
/lgtm
/cc @kubesphere/sig-docs

**Review by weili520 — COMMENTED** [review]
👍

File: locales/en/role.js [hunk 1]
```diff
@@ -55,6 +55,12 @@ module.exports = {
   'Network Policies View': 'Network Policy Viewing',
   'Nodes View': 'Node Viewing',
 
+  PIPELINES_MANAGEMENT: 'Pipeline Management',
+  PIPELINES_VIEW: 'Pipeline Viewing',
+  PIPELINERUNS_MANAGEMENT: 'PipelineRun Management',
```
Comment (Patrick-LuoYu) [inline locales/en/role.js:60]: Pipeline Run Record Management

File: locales/en/role.js [hunk 2]
```diff
@@ -55,6 +55,12 @@ module.exports = {
...
+  PIPELINERUNS_VIEW: 'PipelineRun Viewing',
```
Comment (Patrick-LuoYu) [inline locales/en/role.js:61]: Pipeline Run Record Viewing

File: locales/en/role.js [hunk 3]
```diff
@@ -145,6 +151,15 @@ module.exports = {
   ACCOUNT_ROLE_CREATE_DESC:
     'The role of an account defines the permissions of the account on the KubeSphere platform.',
 
+  PIPELINES_MANAGEMENT_DESC:
+    'Create, edit, and delete DevOps project pipelines.',
+  PIPELINES_VIEW_DESC: 'View DevOps project pipelines and download artifacts.',
+  PIPELINERUNS_MANAGEMENT_DESC:
+    'Create, edit, and delete DevOps project pipelineruns.',
```
Comment (Patrick-LuoYu) [inline locales/en/role.js:158]: View, edit, and delete pipeline run records in a DevOps project.

File: locales/en/role.js [hunk 4]
```diff
@@ -145,6 +151,15 @@ module.exports = {
...
+  PIPELINERUNS_VIEW_DESC: 'View DevOps project pipelineruns.',
```
Comment (Patrick-LuoYu) [inline locales/en/role.js:159]: View pipeline run records in the DevOps project.

**Comment by Patrick-LuoYu** [comment]
The wording needs to be optimized. Please see the comments. Thanks!

**Comment by LinuxSuRen** [comment]
hi @Patrick-LuoYu @harrisonliu5 , sorry for the deply.

**Pull request merged**